package.json

```diff
@@ -103,6 +104,7 @@
     "npm-run-all": "^4.1.2",
     "onchange": "^4.0.0",
     "prettier": "^1.13.7",
+    "react": "^16.4.2",
```

Comment:
Mainly upgraded React here for [the new ref API](https://reactjs.org/docs/refs-and-the-dom.html), but didn't really end up needing it. Still not a bad idea though, seems not to break things.